app/src/main/kotlin/com/mapzen/erasermap/presenter/RoutePresenter.kt

```diff
@@ -6,13 +6,17 @@ import com.mapzen.valhalla.Instruction
 import com.mapzen.valhalla.Route
 
 public interface RoutePresenter {
+    companion object {
+        public val GESTURE_MIN_DELTA = 0.0001f
```

Comment:
Its an arbitrary pixel value to make sure the user is actually trying to drag the map rather than perform another type of gesture.